multiwallet.py

```diff
@@ -465,6 +468,56 @@ def do_generate_seed(self, arg):
             ),
         )
 
+    def do_recover_seed(self, arg):
+        """Recover a seed from shares per SLIP39"""
```

Comment:
I think this should use the phrase Shamir somewhere (in description and/or do_foo method name) as that's what people know this as. There are (terrible) splitting/recovering algorithms that literally cut the phrase into parts (including Ian Coleman's BIP39 which is surprising and unfortunate).